src/getCommands.js

```diff
@@ -8,8 +9,14 @@ const findPlugins = require('./findPlugins');
  * @param  {String}       name Name of the plugin
  * @return {Array|Object}      Plugin's commands
  */
-const getPluginCommands = cwd => name =>
-  require(path.join(cwd, 'node_modules', name));
+const getPluginCommands = cwd => name => {
+  const nested = path.join(cwd, 'node_modules', name);
+
+  /* For rnpm installed locally with flat dependencies structure */
+  const flat = path.join(cwd, '..', name);
```

Comment:
I am generally not happy with that line added because it will also look for plugins in process.cwd()/... which is broken. What we want to is we want to check in __dirname/... (rnpm/node_modules) and __dirname/.../.../ (project node_modules) only.